dev/tasks/tasks.yml

```diff
@@ -245,7 +245,7 @@ tasks:
     ci: azure
     template: conda-recipes/azure.linux.yml
     params:
-      config: linux_64_cuda_compiler_versionNone
+      config: linux_64_c_compiler_version12c_stdlib_version2.12cuda_compilerNonecuda_compiler_versionNonecxx_compiler_version12
     artifacts:
```

Comment:
My guess is that the number and names of artifacts have changed so I need to figure that out for all of these tasks.